src/Language/PureScript/Docs/Types.hs

```diff
@@ -307,9 +307,12 @@ asDeclaration =
               <*> key "fixity" (perhaps asFixity)
               <*> key "info" asDeclarationInfo
 
-asFixity :: Parse PackageError P.Fixity
-asFixity = P.Fixity <$> key "associativity" asAssociativity
-                    <*> key "precedence" asIntegral
+asFixity :: Parse PackageError (P.Fixity, Maybe String)
+asFixity = do
+  fixity <- P.Fixity <$> key "associativity" asAssociativity
+                     <*> key "precedence" asIntegral
+  alias <- key "alias" (perhaps asString)
```

Comment:
So this needs an update to Pursuit, correct? Or is this a backwards-compatible change? Does `perhaps` handle undefined fields?

Maybe we need to bump the min compiler version there. /cc @hdgarrood